ipfx/stimulus.py

```diff
@@ -27,6 +26,8 @@ def has_tag(self, tag, tag_type=None):
 
 class StimulusOntology(object):
 
+    DEFAULT_STIMULUS_ONTOLOGY_FILE = os.path.join(os.path.dirname(__file__), 'defaults/stimulus_ontology.json')
```

Comment:
I wonder if it might make sense to have a separate config file for these kinds of variables?